src/main/java/seedu/address/logic/parser/RemindCommandParser.java

```diff
@@ -10,7 +10,7 @@
 
     /**
      * Parses the given {@code String} of arguments in the context of the DeleteCommand
```

Comment:
is this supposed to be 'in the context of RemindCommand' instead of DeleteCommand?

Reply:
Oh yup. Must have missed it out. I shall go change it now

Reply:
Amended!